lib/generators/dockerfile_generator.rb

```diff
@@ -1450,4 +1452,31 @@ def fly_make_toml
 
     toml
   end
+
+  # if there are multiple production databases defined, allow them all to be
+  # configured via DATABASE_URL.
+  def fix_database_config
+    yaml = IO.read("config/database.yml")
+
+    production = YAML.load(yaml, aliases: true)["production"]
```

Comment:
This actually hard breaks the generate run because not all database.yml files are valid before the erb preprocessing causing this to error.